lib/events.ts

```diff
@@ -31,27 +31,36 @@ const getMixpanel = _.once<any>(() => {
 	});
 });
 
+/**
+ * Mixpanel.com analytics tracking (information on balena CLI usage).
+ *
+ * @param commandSignature A string like, for example:
+ *      "push <applicationOrDevice>"
+ * That's literally so: "applicationOrDevice" is NOT replaced with the actual
+ * application ID or device ID. The purpose is to find out the most / least
+ * used command verbs, so we can focus our development effort where it is most
+ * beneficial to end users.
+ *
+ * The username and command signature are also added as extra context
+ * information in Sentry.io error reporting, for CLI debugging purposes
+ * (mainly unexpected/unhandled exceptions -- see also `lib/errors.ts`).
+ */
 export function trackCommand(commandSignature: string) {
 	const balena = getBalenaSdk();
-	return Promise.props({
+	return Bluebird.props({
 		balenaUrl: balena.settings.get('balenaUrl'),
 		username: balena.auth.whoami().catchReturn(undefined),
 		mixpanel: getMixpanel(),
 	})
 		.then(({ username, balenaUrl, mixpanel }) => {
-			return Promise.try(() => {
-				Raven.mergeContext({
-					user: {
+			return Bluebird.try(() => {
```

Comment:
This `Bluebird.try` can be removed altogether - you're already in a promise chain

Reply:
Indeed, done. 👍